docs/tutorialkit.dev/src/content/docs/guides/using-low-level-apis.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Using low level APIs
+---
+import taskListVideo from "./images/tk-task-list.webm";
+import { HelpButton } from "@components/react-examples/HelpButton";
+import '@tutorialkit/astro/default-theme.css';
+
+TutorialKit exposes low level APIs that authors can utilize to provide highly custom experience in their tutorials.
+
+## Tutorial Store
+
+You can access Tutorial Store by importing the `tutorialkit:store` entrypoint.
+
+```ts
+import tutorialStore from "tutorialkit:store";
+```
+
+The Tutorial Store provides access to current state of the Tutorial, for example the files and their contents.
+
+```ts
+import tutorialStore from "tutorialkit:store";
+
+console.log(tutorialStore.documents.value);
```

Comment:
I think you're trying to be minimal here, but this is not code they should write.

I think we should always suggest using React components and have example that always wait for the lesson to be loaded via `useStore(tutorialStore.lessonFullyLoaded)`

(unless they run after a user action in which case they can ignore the wait).